src/NMoneys/Currency.cs

```diff
@@ -163,33 +163,46 @@ private void setAllFields(CurrencyIsoCode isoCode, string englishName, string na
 			IsObsolete = isObsolete;
 			Entity = entity;
 
-			FormatInfo = NumberFormatInfo.ReadOnly(new NumberFormatInfo
-			{
-				CurrencySymbol = symbol,
-				CurrencyDecimalDigits = significantDecimalDigits,
-				CurrencyDecimalSeparator = decimalSeparator,
-				CurrencyGroupSeparator = groupSeparator,
-				CurrencyGroupSizes = groupSizes,
-				CurrencyPositivePattern = positivePattern,
-				CurrencyNegativePattern = negativePattern,
-				NumberDecimalDigits = significantDecimalDigits,
-				NumberDecimalSeparator = decimalSeparator,
-				NumberGroupSeparator = groupSeparator,
-				NumberGroupSizes = groupSizes,
-				NumberNegativePattern = negativePattern.TranslateNegativePattern(),
-			});
-		}
-
-		#endregion
-
-		#region static shortcuts & cache initialization
-
-		// random list of static accessors based on currency exchange most used currencies
-
-		/// <summary>
-		/// Australia Dollars
-		/// </summary>
-		public static readonly Currency Aud;
+            var formatInfo = new NumberFormatInfo

```

Comment:
Shouldn't we still wrap in NumberFormatInfo.ReadOnly, except when isoCode = ZZZ?